docs/decisions/0001-open-analytics.rst

```diff
@@ -0,0 +1,110 @@
+actively supported around 2018. The goal at the time was to eventually replace Insights, but
+that project was never picked up. While the community has done some recent work circa 2022 to update
+Insights, it was never a widely adopted solution for the wider community. Interviews with the community
+have revealed a few reasons why:
+
+- It is complicated, comprising 6 Github repositories, many pieces of infrastructure, and requiring
+  knowledge of several domain specific technologies to configure (Pandas, Hive, Sqoop, Hadoop,
+  Luigi, etc.)
+- It is expensive to run, and is in many ways specific to Amazon Web Services technologies
+- Turnaround time for data refreshes are on the order of a day or more in most cases
+- Documentation is out of date, further complicating any new adoption or alternative deployments
+- Discrepancies between Insights data calculations and data displayed in Studio have caused confusion
+
+Architectural decisions made post-Insights and new technologies have changed the analytics
+landscape, unlocking the ability to deliver analytical and operational data and display it in
+near-real time on commodity hardware with much simpler configuration and deployment. Additionally
+we have a wide variety of use cases in the Open edX community with differing requirements for
+privacy, scalability, budget, and expertise.
+
+Decision
+********
+
+We will create the Open Analytics Reference System (OARS) that combines existing open-source projects
+into a preconfigured bundle that can be easily deployed using Tutor.
+
+These projects will include:
+
+- A Learning Record Store (LRS)
+- A service to transform tracking log events into an open standard
+- An analytic database
+- A data visualization and dashboard tool with a data export API
+- Code and configuration to tie these tools together, as well as rich reports that work against the
+  default configuration
+
+The guiding principals for technology selection are:
+
+- Based on open standards and open source
+- Hosting service agnostic
+- Inexpensive to run
+- Able to support near-real-time data where possible
+- Require little specialized knowledge to set up and maintain
+- Be extensible for a variety of common use cases not covered by the default configuration
+
+This system will:
+
+- Transform existing Open edX tracking log events into an open standard format
+- Store them using a standards-compliant learning record store
+- Present a user interface of data visualizations secured via single-sign-on against the LMS
+- Allow download of report data for those with permissions to view it
+- Provide a secure API for integrations with other tools or data viewing methods
+- Be configured to send data de-identified by default, with an option for to send events
+  as identified for use cases that require them such as connected users across different LMSs
+  - This de-identification is imperfect and cannot prevent learners from, for example, putting
```

Comment:
A little more why than how would be valuable here.  Could you start with "This system will endeavor to be privacy preserving.  As such, we will de-identify learner data by default.  We acknowledge that supporting learner may at time require linking data to identity.  Whenever we consider the use of identity data we will take the most conservative approach, focusing on preserving learner privacy and data ownership.

Reply:
👍 to @e0d 's suggestion of "more why than how". And because we're being brief with everything else, I think it's ok to be brief here too:

> "This system will endeavor to be privacy preserving, by de-identifying learner data by default and focusing on respecting learner privacy and data ownership when storing identity data."

Reply:
I have been wavering on whether privacy requires its own ADR, and now I think it will. For now @pomegranited 's suggestion is closer to the high level spirit of this doc, and once more details are hammered out with identifiers I'll be able to write a more fleshed out ADR with guidelines on privacy and data lifecycle. Does that work for you @e0d ?